packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

```diff
@@ -162,257 +163,334 @@ Continue with the OIN Wizard and specify your supported SSO flows:
+> * Click **Initialize Tester** if you're using the OIN Submission Tester for the first time.
+> * Click **Refresh Tester session** for a new test session if the OIN Submission Tester session expired.
 
-Test case preconditions:
+The OIN Submission Tester includes the mechanism to test the following flows:
 
-* App integration supports SP flow
-* App integration supports JIT provisioning
-* [App integration instance was generated in the OIN Wizard](#generate-the-instance-for-testing) and sign-on options are configured
-* The [Sign in with the SP flow](#how-to-test-an-sp-flow) test case was executed successfully
-* A new test user profile that isn't in Okta or in your app
+* IdP flow
+* SP flow
+* Just-In-Time (JIT) provisioning (with IdP flow)
+* Just-In-Time (JIT) provisioning (with SP flow)
 
-To test JIT provisioning with the SP flow:
+> **Note:** The **JIT provisioning (with SP flow)** test case appears in the OIN Submission Tester if your integration supports JIT and only the SP flow. If your integration supports JIT, IdP, and SP flows, then a successful **JIT provisioning (with IdP flow)** test is sufficient for submission.
 
-1. As an admin user, verify that the test user doesn't exist in your Okta org.
-1. As an admin user, verify that the test user doesn't exist in the app. Sign in to your app and verify that there's no user with the same unique attributes as your new test user.
-1. As an admin user, sign in to your Okta org.
-1. Go to **Directory** > **People** and add the new test user in Okta. See [Add users manually](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-add-users.htm) for complete instructions.
-1. Go to **Applications** > **Applications** > your app integration instance in the Admin Console.
-1. From your app integration instance, click the **Assignments** tab.
-1. Click **Assign** > **Assign to People**.
-1. Find the name of the new test user and click **Assign** next to their name. A dialog box appears with the title **Assign {app-name} to People**.
-1. Click **Save and Go Back**.
-1. On the **People** page, click **Done**.
-1. Open a new incognito window in your browser.
-1. Go to the app sign-in page directly from the browser URL address field (for example: `https://berryfarm.example.org/strawberry/signin`). The browser redirects you to Okta for authentication.
-1. Sign in with Okta credentials for the new test user that was assigned to the app integration.
-1. Confirm that you can sign in to the app.
-1. Sign out of your app.
-1. Verify that you're able to sign out and are redirected to the sign-in page.
-1. Verify that the new user was created in your app with supported attributes passed from the Okta profile.
+The test cases for these flows appear in the **Test integration using the OIN Submission Tester** section depending on your OIN Wizard [test information](#test-information).
 
-### Failed SSO tests
+See [Run test](#run-tests) to run each of the test cases.
```

Comment:
I made this a note because it's buried in a sea of text otherwise :) 

```suggestion
> **Note:** See [Run test](#run-tests) for the steps on how to run each test case.
```

Reply:
Thanks! I'll update this with another commit.